src/features/callAssignments/components/CallAssignmentStatusCards.tsx

```diff
@@ -0,0 +1,114 @@
+import { useIntl } from 'react-intl';
+import { Card, Grid, List } from '@material-ui/core';
+
+import { CallAssignmentStats } from '../apiTypes';
+import StatusSectionHeader from './StatusSectionHeader';
+import StatusSectionItem from './StatusSectionItem';
+
+//TO-DO: Replace with theme colors
+const ORANGE = 'rgba(245, 124, 0, 1)';
```

Comment:
Maybe use the theme (i.e. add theme properties) for these already, and then we can just replace their values?